app/models/cherrypick/task/pick_by_micro_litre.rb

```diff
@@ -5,11 +5,6 @@
 # Copyright (C) 2007-2011,2012,2015 Genome Research Ltd.
 
 module Cherrypick::Task::PickByMicroLitre
-  def pick_by_micro_litre(*args)
```

Comment:
This code was only used by the cherrypick by pulldown pipeline, which has been deprecated, and the associated code removed. Same for the similar methods in the next few files.